e2e/helper/connect.ts

```diff
@@ -0,0 +1,44 @@
+import { Page, expect } from "@playwright/test";
+
+export const clickConnectButton = async (page: Page) => {
+  const connectButton = page.getByRole("button", {
+    name: "Connect to BTC",
+  });
+  await connectButton.click();
+};
+
+export const acceptTermsAndConditions = async (page: Page) => {
+  const termsCheckbox = page
+    .locator("label")
+    .filter({ hasText: "I certify that I have read" });
+
+  const inscriptionsCheckbox = page
+    .locator("label")
+    .filter({ hasText: "I certify that there are no" });
+
+  const hwCheckbox = page
+    .locator("label")
+    .filter({ hasText: "I acknowledge that Keystone via QR code" });
```

Comment:
This should be done in one place and our app should consume it the same way e2e tests do:
```mermaid
graph TD
    A[Text Constants] --> B[App]
    A[Text Constants] --> C[E2E Tests]
```
Extracted to https://github.com/babylonlabs-io/simple-staking/issues/136, otherwise this PR will touch the `simple-staking` app, and not the initial e2e testing part

If we just extract this as constants and do not consume this inside the app, then we will still have the problem of 2 sources of truth